sidebars.js

```diff
@@ -149,6 +148,14 @@ module.exports = {
       },
       "operate-a-stake-pool/marketing-stake-pool",
     ],
+    "Stake Pool Guide (in Work)": [
+      "stake-pool-guide/introduction",
+      "stake-pool-guide/pre-requisites",
+      "stake-pool-guide/install-configure-node",
+      "stake-pool-guide/register-pool",
+      "stake-pool-guide/pool-maintainance",
+      "stake-pool-guide/tools-links",
```

Comment:
Might be good to remove (in Work) from title on front page visibility, and add `stake-pool-guide/best-practices` as a new page (even if empty) as placeholder for #42